lms/templates/footer.html

```diff
@@ -16,92 +16,85 @@
   footer = get_footer(is_secure=request.is_secure())
   footer['copyright'] = _(u"\u00A9 {year} Massachusetts Institute of Technology").format(year=datetime.now().year)
   footer['navigation_links'] = [link for link in footer['navigation_links'] if link.get("name")!="help-center"]
-%>
-<%namespace name='static' file='static_content.html'/>
 
-% if uses_bootstrap:
-  <div class="container-fluid wrapper-footer">
-    <footer>
-      <div class="row">
-        <div class="col-md-9">
-          <nav class="navbar site-nav navbar-expand-sm" aria-label="${_('About')}">
-            <ul class="navbar-nav">
-              % for item_num, link in enumerate(footer['navigation_links'], start=1):
-                <li class="nav-item">
-                  <a class="nav-link" href="${link['url']}">${link['title']}</a>
-                </li>
-              % endfor
-            </ul>
-          </nav>
+  icp_license = getattr(settings,'ICP_LICENSE', False)
+  icp_markup = u" | {icp}".format(icp=icp_license) if icp_license else ""
 
-          ## Site operators: Please do not remove this paragraph! This attributes back to edX and makes your acknowledgement of edX's trademarks clear.
-          <p class="copyright">${footer['copyright']} ${u" | {icp}".format(icp=getattr(settings,'ICP_LICENSE')) if getattr(settings,'ICP_LICENSE',False) else ""}</p>
+  footer_links = {
+    "about": {"url": None, "title": None},
+    "terms_of_service": {"url": None, "title": None},
+    "accessibility_policy": {"url": None, "title": None},
+    "contact": {"url": None, "title": None}
+  }
+  for link in footer['navigation_links'] + footer['legal_links']:
+      url = link.get('url')
+      name = link.get('name')
+      if not url or name not in footer_links.keys():
+          continue
+      if name == "about":
+          link["title"] = "About Us"
```

Comment:
We can make this simpler. Titles can be added as per above comment and we jjust need to add the URL here.